Telerik.JustMock.Tests/MockResetFixture.cs

```diff
@@ -0,0 +1,120 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+#region JustMock Test Attributes
+#if NUNIT
```

Comment:
@MariaDch , you can simplify the code as follows:

```csharp
       ...

        [ClassInitialize]
#if NUNIT && NUNIT3
        public static void ClassInitialize()
#else
        public static void ClassInitialize(TestContext ctx)
#endif
        {
            staticFoo = Mock.Create<Foo>();
            Mock.Arrange(() => staticFoo.FooNotImplemented());
        }
        ...
```

Reply:
@ivo-stoilov As discussed offline we will keep this unchanged to ensure that all tests are functioning as expected.